### DISCUSSION TITLE
Create a stroke around selection

### DISCUSSION THREAD

**Discussion by skullb0x** [photopea/photopea#6912] [Q&A]
Hi!

Love Photopea, I have been using Photoshop a lot and Photopea fills all my need so far except that I really miss the feature to draw a line around my selection. In Photoshop you could make whatever selection and then right click and get the option to stroke and select the width of the line to draw. Really helpful when drawing pixel art and you want a perfect pixelated border around your pixelated shape. Is this feature maybe already in Photopea and have I missed it?

Edit: it is not the same as Layer style stroke, since that draws an outline around or insde the drawing, also this selection stroke keeps the pixel-perfect shape of the selection which is neat.

**Comment by photopea** [comment]
It should be avaliable in the right-click menu now.
